.github/workflows/ci.yml

```diff
@@ -97,6 +97,10 @@ jobs:
           pdm info
       - name: Run Tests
         run: pdm run pytest -n auto --cov=pdm --cov-config=pyproject.toml --cov-report=xml tests
+      - name: Run Tests Without MsgPack
+        run: |
+          pdm run pip uninstall -y msgpack
```

Comment:
Previously dependencies are installed with `pdm install -Gtest` (L96) which should not include msgpack

You just need to do `pdm install -G msgpack` here.